crawler/helpers.py

```diff
@@ -175,11 +171,11 @@ def map_mongo_doc_to_sql_columns(doc) -> Dict[str, Any]:
             Dict[str, str] -- Dictionary of MySQL versions of fields
     """
     value = map_mongo_to_sql_common(doc)
-    value[MLWH_CREATED_AT] = datetime.strftime(doc[FIELD_CREATED_AT], MYSQL_DATETIME_FORMAT)
-    value[MLWH_UPDATED_AT] = datetime.strftime(doc[FIELD_UPDATED_AT], MYSQL_DATETIME_FORMAT)
+    value[MLWH_CREATED_AT] = doc[FIELD_CREATED_AT]
```

Comment:
So here the doc has a datetime value, so you can just pass that to the MLWH as is? great